plugin.video.vrt.nu/resources/lib/helperobjects/streamurls.py

```diff
@@ -11,7 +11,7 @@ def __init__(self, stream_url, subtitle_url=None, license_key=None, use_inputstr
         self._stream_url = stream_url
         self._subtitle_url = subtitle_url
         self._license_key = license_key
-        self._use_inputstream_adaptive = use_inputstream_adaptive
+        self.use_inputstream_adaptive = use_inputstream_adaptive
```

Comment:
is this correct? Shouldn't this be like _use_inputstream_adaptive  this? like the others

Reply:
nvm saw where it is comming from and corrected it